x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table.tsx

```diff
@@ -93,14 +96,17 @@ export const SpikeAnalysisTable: FC<SpikeAnalysisTableProps> = ({
           </>
         </EuiToolTip>
       ),
-      render: (_, { histogram, fieldName, fieldValue }) => {
-        return histogram ? (
-          <MiniHistogram chartData={histogram} label={`${fieldName}:${fieldValue}`} />
-        ) : null;
-      },
+      render: (_, { histogram, fieldName, fieldValue }) => (
```

Comment:
We need to use the EUI theme to set the color of the selected row as the contrast in dark theme is poor:

<img width="775" alt="image" src="https://user-images.githubusercontent.com/7405507/181258893-0d689807-522b-4198-9b57-8060bf88d84b.png">

Reply:
Fixed in [1d2013e](https://github.com/elastic/kibana/pull/137266/commits/1d2013ebe14a8a40a15e66767376ebf840fa05c2).

<img width="1015" alt="image" src="https://user-images.githubusercontent.com/230104/181260958-253b6851-6880-447c-a7fa-e8a31fb5e5e9.png">